igor-core/src/main/java/com/netflix/spinnaker/igor/build/model/GenericBuild.java

```diff
@@ -26,6 +26,8 @@
 import lombok.NoArgsConstructor;
 
 @Data
+@NoArgsConstructor
```

Comment:
hi @robzienert do you have any concerns with adding these annotations here?

Reply:
No, but I'm not sure it's necessary? `@Data` already implicitly adds `@RequiredArgsConstructor`.

Reply:
Oh. Nvm.